test_requests/island_routes.txt

```diff
@@ -0,0 +1,4 @@
+-j '{"locations":[{"lat":-33.4280388615334,"lon":-70.54088473320007},{"lat":-33.4268390099302,"lon":-70.53765535354614}],"costing":"pedestrian"}'
```

Comment:
i couldnt find any good island in utrecht so i made a island routes file instead

Reply:
You won't believe it but there is a Prison Island in Utrecht: http://prisonisland.se/en/utrecht/ -- not disconnected from the road network though 😆. 

Anyway I was able to trigger the bug on master with these coordinates in Utrecht with bike costing `5.011605999215277,52.114736949245156;5.020381755701578,52.10586968702347` if that helps.